### ISSUE TITLE
RectLabel crashes at launch time after xml object was modified

### ISSUE THREAD

**Issue by leoburgy** [ryouchinsa/Rectlabel-support#52]
In the middle of an annotation, I renamed an object's name (Capitalise). When exporting the mask, the images where the objects still had lower case label did not contain any object. After figuring out the inconsistency, I capitalised all the markup in the xml and since then RectLabel cannot be open anymore. I tried to use the original xml files and restarted everything. Nothing worked so far.

Thank you very much if you could provide a quick fix :)

**Comment by ryouchinsa** [comment]
Thanks for writing the issue.

When you export the mask images, each label name should include one of the object names on the settings dialog.
At this moment, we are distinguishing between uppercase and lowercase letters.
We can change so that we do not distinguish between uppercase and lowercase letters.

For the crash, could you send us the image and xml file?
We would fix the problem.

Best regards,
Ryo

**Comment by leoburgy** [comment]
Thanks for your prompt reply.

I'll try to decouple the nested issues I've experienced so far and which I solved:

1. It happens that one of my xml files was inadvertently emptied; this caused RectLabel to crash at launch. Once the I removed the empty xml, RectLabel launched again seamlessly.
2. The capitalisation of objects is only a problem when exporting the accumulated mask into a single png file.

Thank you for the support.
